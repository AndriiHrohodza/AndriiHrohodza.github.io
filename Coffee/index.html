<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="css/normalize.css">
    <link rel="stylesheet" type="text/css" href="css/styleCoffee.css">
    <title>Coffe</title>
  </head>
  <body>
    <main class="main">
      <div class="container-main">
        <div class="header clearfix">
          <div class="header__return-call">
            <p class="phone"><strong>+7 (908) 710 59 49</strong></p>
            <a href="">Обратный звонок</a>
          </div>
          <div class="header__street">
            <p>г. Сыктывкар, <br>
              ул. Громова, д.81
            </p>
          </div>
          <div class="header__logo">
            <img src="img/header/logo.svg" alt="logo">
          </div>
          <div class="header__button">
            <button class="btn">Магазин</button>
          </div>
        </div>
      </div>
    </main>
    







    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <!-- JS main-->
    <script src="js/jsCoffee.js"></script>
  </body>
</html>