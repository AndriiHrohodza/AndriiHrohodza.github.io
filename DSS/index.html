<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- Normalize -->
    <link rel="stylesheet" href="css/normalize.css">
    <!-- Animate -->
    <link rel="stylesheet" href="css/animation.css">
    <!-- Main Style -->
    <link rel="stylesheet" href="css/styleDSS.css">
    <script src="https://code.jquery.com/jquery-3.4.1.js"></script>
    <title>DSS</title>
  </head>
  <body>

    <div class="wrap">
      <div class="container-wrap">
        <!--Header-->
        <header class="header">
          <div class="header__content">
            <div class="header__section-logo">
              <div class="header-item logo">
                <a href="#">B2B Cloud</a>
              </div>
            </div>
            <div class="header__section-nav">
              <div class="header-item about">
                <a href="#">About</a>
              </div>
              <div class="header-item services">
                <a href="#">Services</a>
              </div>
              <div class="header-item product">
                <a href="#">Product</a>
              </div>
              <div class="header-item form">
                <a href="#">Form</a>
              </div>
            </div>
          </div>
        </header>
        <!--Main-->
        <main class="main">
          <section class="primary wow fadeInUp">
            <p>Rent project-based rescurces</p>
            <h2>Explore, compare and contract <br> various types of resources for <br> middle term projects in one place.</h2>
            <button class="btn"><a class="active" href="#">Find specialist +</a></button>
          </section>
          <div class="scroll">
            <div class="scroll-down">
              <div class="scroll-button">
                <a href="#wrap-main" >Scroll Down</a>
              </div>  
            </div> 
          </div>
        </main>
      </div>
    </div>
    <div id="wrap-main" class="wrap-main">
      <section class="company-description">
        <div class="container">
          <div class="description-content">
            <div class="description-content__img">
              <img src="img/main/company-description/background.jpg" alt="">
            </div>
            <div class="description-content__text wow fadeInUp">
              <h3>Who we are.</h3>
              <p class="cinetype">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
              <p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt.</p>
              <p>Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem.</p>
            </div>
          </div>
        </div>
      </section>
      <section class="features clearfix">
        <div class="features-content">
          <div class="features-content__items">
            <div class="items-img">
              <img src="img/main/features/1.png" alt="">
            </div>
            <div class="items-text">
              <div>
                <h3>Describe your project timeline</h3>
                <p>Describe your project timeline, place and business terms and choose what type of specialists do you need.</p>
              </div>
            </div>
          </div>
          <div class="features-content__items">
            <div class="items-img">
              <img src="img/main/features/2.svg" alt="">
            </div>
            <div class="items-text">
              <div>
                <h3>Describe your project timeline</h3>
                <p>Describe your project timeline, place and business terms and choose what type of specialists do you need.</p>
              </div>
            </div>
          </div>
          <div class="features-content__items">
            <div class="items-img">
              <img src="img/main/features/3.svg" alt="">
            </div>
            <div class="items-text">
              <div>
                <h3>Describe your project timeline</h3>
                <p>Describe your project timeline, place and business terms and choose what type of specialists do you need.</p>
              </div>
            </div>
          </div>
        </div>
      </section>
    </div>

    <footer class="footer clearfix">
      <h1>Nice work!</h1>
    </footer>
	
		




    <!-- jQuery -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <!-- Scroll Magic -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/ScrollMagic/2.0.7/ScrollMagic.min.js" integrity="sha256-2p2tRZlPowp3P/04Pw2rqVCSbhyV/IB7ZEVUglrDS/c=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/ScrollMagic/2.0.7/plugins/debug.addIndicators.min.js" integrity="sha256-XBVALc3QsDSQiDtiaBwNZ2rxCJTHzlUhhlDQ84J8bw8=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/ScrollMagic/2.0.7/plugins/animation.gsap.min.js" integrity="sha256-+9YNuItWuR4sbqeaNiJOxG0BvptYz4fbUXbIZoH5Jwo=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/2.1.3/TweenLite.min.js" integrity="sha256-VV47uJSoHZUeiBcCs3FcBOQLMn++yeG/zqZvaUkvGZM=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/2.1.3/TimelineLite.min.js" integrity="sha256-nuhNsfXzBFR6G1lKP8bK77dakkQDqdHcQ4OCFZvk6Qo=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/2.1.3/plugins/CSSPlugin.min.js" integrity="sha256-LBjlnpPrM6Aig8LDFc9PJctPHLGUc6RaUvnmXE4hV5Y=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/2.1.3/plugins/BezierPlugin.min.js" integrity="sha256-5jxCMRC1PYU02qJn+fj+DPuxcQZCh0DR8GRwi4iKoRc=" crossorigin="anonymous"></script>
    <!-- WOW JS -->
    <script src="js/wow.min.js"></script>
    <script>
      new WOW().init();
    </script>
    <!-- Main JS -->
    <script src="js/jsDSS.js"></script>
    
 		<!--<script type="text/javascript">
 			$(document).on('scroll', function(){
 				$('h1').css("left", Math.max(500 - 0.35*window:scrollY) + "px");
 			})
 		</script>-->
  </body>
</html>
