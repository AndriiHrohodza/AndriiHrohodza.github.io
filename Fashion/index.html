<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Проект 2</title>
	<link rel="stylesheet" type="text/css" href="css/style.css">
</head>
<body>
	<main class="main">
		<div class="container">
			<div class="header">
					<div class="header-logo">
						<a href=""><img src="images/Vector.png" alt="logo"></a>
					</div>
					<ul class="header-nav">
						<li><a href="#">Albums</a></li>
						<li><a href="#">Eilish</a></li>
						<li><a href="#">Merch</a></li>
						<li class="header-nav-right"><a href="#">PlayList</a></li> 
						<li class="header-nav-menu"><a href="#"><img src="images/Group.svg" alt="menu"></a></li>
					</ul>
			</div>
			<div class="content">
				<div class="textleft">
					<h1>dont <br> smile</h1>
					<p class="atme"> at me </p>
				</div>
				<div class="card">
					<div class="card-item" ></div>
				</div>
				<div class="textright">
					<ol>
						<li>
							1. copycat
						</li>
						<li>
							2. idontwannabe ...
						</li>
						<li>
							3. my boy
						</li>
						<li>
							4. watch
						</li>
						<li>
							5. patry favor
						</li>
						<li>
							6. bellyache
						</li>
						<li>
							7. ocean eyes
						</li>
					</ol>
				</div>
			</div>
			<div class="footer">
				<div class="footer-start">
					<a href="#">Start</a>
				</div>
				<div class="footer-line">
					<div class="footer-line-1px"></div>
					<div class="footer-line-menu">
						<a class="menu" href="#">Menu</a>
					</div>
				</div>
			</div>
		</div>
	</main>

<script src="js/main.js"></script>
</body>
</html>