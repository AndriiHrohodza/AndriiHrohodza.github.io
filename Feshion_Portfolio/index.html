<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="css/normalize.css">
    <link rel="stylesheet" href="css/styleFeshionPortfolio.css">
    <title>Feshion Portfolio</title>
      
  </head>
  <body>
    

    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <!-- Include all compiled plugins (below), or include individual files as needed -->
    <script src="js/bootstrap.min.js"></script>
  </body>
</html>