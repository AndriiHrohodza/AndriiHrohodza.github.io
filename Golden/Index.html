<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Верстка</title>
	<link rel="stylesheet" href="style.css">
</head>
<body>
	<div id="wrapper">
		<div id="header">
			<div class="container blockquote">
				<div class="logo">
					<a href=""><img src="image/Logo-Treehouse.png" alt="logo"></a>
				</div>
				<ul class="nav">
					<li><a href="#">Home</a></li>
					<li><a href="#">Services</a></li>
					<li><a href="#">Portfolio</a></li>
					<li><a href="#">About</a></li> 
					<li><a href="#">Contact</a></li>
				</ul>
			</div>
		</div>
		<div id="home">
			<div class="container"></div>
				<h3>Welcome To Our Studio!</h3>
				<h1>It’s nice to Meet you</h1>
				<button class="btn">Tell Me More</button>
		</div>
		<div id="servises" class="box">
			<div class="container">
				<h2 class="title">Services</h2>
					<p class="desc">Proin iaculis purus consequat sem cure. 
					</p>
					<div class="services-wrap blockquote">
						<div class="services-ecommers basket">
							<h5 class="title">E-Commerce</h5>
							<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipitaenean rhoncus posuere odio in tincidunt.</p>
						</div>
						<div class="services-ecommers laptop">
							<h5 class="title">Responsive Web</h5>
							<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipitaenean rhoncus posuere odio in tincidunt.</p>
						</div>
						<div class="services-ecommers lock">
							<h5 class="title">Web Security</h5>
							<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipitaenean rhoncus posuere odio in tincidunt.</p>
						</div>
					</div>
			</div>
		</div>
		<div id="portfolio" class="box">
			<div class="container">
				<h2 class="title">Our portfolio</h2>
					<p class="desc">Proin iaculis purus consequat sem cure.</p>
					<div class="portfolio-wrap blockquote">
						<div class="portfolio-item">
							<a href=""><img src="image/Image.jpg" alt=""></a>
							<div class="p-item-footer">
								<h5 class="title up">Ebony & Ivory</h5>
								<span>Branding</span>
							</div>
						</div>
						<div class="portfolio-item">
							<a href=""><img src="image/Image2.jpg" alt=""></a>
							<div class="p-item-footer">
								<h5 class="title up">Smart Stationary</h5>
								<span>Print Design</span>
							</div>
						</div>
						<div class="portfolio-item">
							<a href=""><img src="image/Image3.jpg" alt=""></a>
							<div class="p-item-footer">
								<h5 class="title up">Clever Poster</h5>
								<span>Print Design</span>
							</div>
						</div>
						<div class="portfolio-wrap blockquote">
						<div class="portfolio-item">
							<a href=""><img src="image/Image4.jpg" alt=""></a>
							<div class="p-item-footer">
								<h5 class="title up">Ebony & Ivory</h5>
								<span>Branding</span>
							</div>
						</div>
						<div class="portfolio-item">
							<a href=""><img src="image/Image5.jpg" alt=""></a>
							<div class="p-item-footer">
								<h5 class="title up">Smart Stationary</h5>
								<span>Print Design</span>
							</div>
						</div>
						<div class="portfolio-item">
							<a href=""><img src="image/Image6.jpg" alt=""></a>
							<div class="p-item-footer">
								<h5 class="title up">Clever Poster</h5>
								<span>Print Design</span>
							</div>
						</div>
					</div>
				</div>
			</div>
		</div>
		<div id="about" name="about" class="box">
			<div class="container">
				<h2 class="title">About Us</h2>
					<p class="desc">Proin iaculis purus consequat sem cure.</p>
				<div class="about-wrap">
					<div class="about-item clearfix">
						<div class="about-text">
							<h5 class="title">JULY 2010<br/> Our Humble Beginnings</h5>
							<p class="q">
							Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt. Proin 
							iaculis purus consequat sem cure digni 
							ssim. Donec porttitora entum suscipit.</p>
						</div>
						<div class="about-img">
							<img src="image/Image.png" alt="">
						</div>	
					</div>
					<div class="about-item1 clearfix">
						<div class="about-text1">
							<h5 class="title1">JANUARY 2011 <br/>Facing Startup Battles</h5>
							<p class="qq">
							Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt. Proin 
							iaculis purus consequat sem cure digni 
							ssim. Donec porttitora entum suscipit.</p>
						</div>
						<div class="about-img1">
							<img src="image/Image1.png" alt="">
						</div>	
					</div>
					<div class="about-item2 clearfix">
						<div class="about-text">
							<h5 class="title">DECEMBER 2012<br/>Enter The Dark Days</h5>
							<p class="q">
							Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt. Proin 
							iaculis purus consequat sem cure digni 
							ssim. Donec porttitora entum suscipit.</p>
						</div>
						<div class="about-img">
							<img src="image/Image3.png" alt="">
						</div>	
					</div>
					<div class="about-item1 clearfix">
						<div class="about-text1">
							<h5 class="title1">FEBRUARY 2014<br/> Our Triumph</h5>
							<p class="qq">
							Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt. Proin 
							iaculis purus consequat sem cure digni 
							ssim. Donec porttitora entum suscipit.</p>
						</div>
						<div class="about-img1">
							<img src="image/Image4.png" alt="">
						</div>	
					</div>
					<div class="about-item3 clearfix">
						<div class="about-container">
							<img src="image/Container.png" alt="">
						</div>
						<div class="about-container1">
							<h5 class="title-container">
								<a href="#" class="about-link">Our</a><br/> <a href="#" class="about-link">Story</a><br/> <a href="#" class="about-link">Continues</a><br/><a href="#" class="about-link">...</a>
							</h5>
						</div>	
					</div>
				</div>
			</div>
		</div>
		<div id="team" class="box">
			<div class="container">
				<h2 class="title">our amazing team</h2>
					<p class="desc">Proin iaculis purus consequat sem cure.</p>
					<div class="team-wrap cleafix">
						<div class="team-item">
							<img src="image/Container1.png">
							<h5 class="title2">Kimberly Thompson</h5>
							<p class="cat">Marketer</p>
							<div class="social">
								<a href="#" class="ic-twit"></a>
								<a href="#" class="ic-fb"></a>
								<a href="#" class="ic-pn"></a>
							</div>
						</div>
					</div>
					<div class="team-wrap cleafix">
						<div class="team-item clearfix">
							<img src="image/Container1.png">
							<h5 class="title2">Rico Massimo</h5>
							<p class="cat">Coder</p>
							<div class="social">
								<a href="#" class="ic-twit"></a>
								<a href="#" class="ic-fb"></a>
								<a href="#" class="ic-gl"></a>
							</div>
						</div>
					</div>
					<div class="team-wrap cleafix">
						<div class="team-item clearfix">
							<img src="image/Container1.png">
							<h5 class="title2">Uku Mason</h5>
							<p class="cat">Graphic Designer</p>
							<div class="social">
								<a href="#" class="ic-fb"></a>
								<a href="#" class="ic-pn"></a>
							</div>
						</div>
					</div>
			<div class="team-info clearfix">
				<div class="container">
				<p class="desc iqq">Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit <br/>  
					aenean rhoncus posuere odio in tincidunt proin culis.</p>
				</div>
			</div>
			</div>
		</div>
			<div class="brands">
				<div class="container">
					<a href="#"><img src="image/Envato.jpg" alt=""></a>
					<a href="#"><img src="image/WordPress.jpg" alt=""></a>
					<a href="#"><img src="image/Tuts+.jpg" alt=""></a>
					<a href="#"><img src="image/Microlancer copy.jpg" alt=""></a>
				</div>
			</div>
			<div id="contact" name="contact" class="box">
				<div class="container">
				<h2 class="title">contact us</h2>
					<p class="desc">Proin iaculis purus consequat sem cure.</p>
					<form action="#" id="messege">
						<div class="clearfix">
							<div class="left">
								<input type="text" placeholder="your Name *">
								<input type="text" placeholder="your e-mail *">
								<input type="text" placeholder="subject">
							</div>
							<div class="right">
								<textarea cols="0" rows="0" placeholder="Your message *">
								</textarea>
							</div>
						</div>
						<input type="submit" value="Send Message" class="btn">
					</form>
				</div>
			</div>
			<div id="copyright">
				<div class="container">
					<p class="copy">&copy; Copyright 2014 FreebiesXpress.com</p>
					<div class="social">
						<a href="#" class="ic-twit www"></a>
						<a href="#" class="ic-fb www "></a>
						<a href="#" class="ic-gl www"></a>
						<a href="#" class="ic-pn www"></a>
					</div>
				</div>
			</div>		
	</div>
</body>
</html>